frontend/src/metabase/query_builder/components/DataSelector/DataSelectorTablePicker/DataSelectorTablePicker.tsx

```diff
@@ -98,6 +102,16 @@ const DataSelectorTablePicker = ({
     const renderItemIcon = ({ table }: { table: Table }) =>
       table ? <Icon name="table" /> : null;
 
+    const renderItemExtra = ({ table }: { table: Table }) => {
+      return (
+        <DataSelectorTablePickerLoader>
```

Comment:
Double checking - should we omit rendering the extra item instead of showing empty  `DataSelectorTablePickerLoader`?

Reply:
I've removed DataSelectorTablePickerLoader because it wasn't needed.